src/drivers/power_monitor/ina226/ina226.cpp

```diff
@@ -174,22 +173,15 @@ INA226::force_init()
 int
 INA226::probe()
 {
-	int value = read(INA226_MFG_ID);
-
-	if (value < 0) {
-		perf_count(_comms_errors);
-	}
+	int16_t value{0};
+	read(INA226_MFG_ID, value);
 
 	if (value != INA226_MFG_ID_TI) {
```

Comment:
Can you check the return value? I know it works the way it is implemented, but is just more robust.
```suggestion
	if (read(INA226_MFG_ID, value) != 0 || value != INA226_MFG_ID_TI) {
```

Reply:
Makes sense, before it was only increasing the `_comms_errors` again for the same I2C error. That's why I removed the check. But obviously it makes more sense to fail early when the IDs cannot be read which is probably the expected case when the external power module is not attached 👍